app/controllers/polls.rb

```diff
@@ -3,6 +3,11 @@
   erb :'polls/index'
 end
 
+get'/polls/:poll_id/responses' do
+  @poll = Poll.find(params[:poll_id])
```

Comment:
I've seen Sam and Steve make this comment so I'll just leave it here. If you have a long parameter name in your route, you can use the `get '/route/:very_long_parameter_name do |n|` pattern to shorten the names within the route definition. For example

```ruby
get '/polls/:poll_id/responses' do |id|
  @poll = Poll.find(id)
end
```

Note that not only did we get to shorten the name from `:poll_id` to `:id`, but now we don't even have to look in the params hash, as that is done automatically.